### DISCUSSION TITLE
About the "flat" option on EAN and UPC

### DISCUSSION THREAD

**Discussion by TheBlindHawk** [lindell/JsBarcode#453] [Q&A] [answered]
What changes does the "flat" option provide when creating EAN/UPC barcodes?

I tried checking the READMEs but couldn't find any explanations [here](https://github.com/lindell/JsBarcode/wiki/Options#flat).
Also when I tried the demo there were no visual differences?
I am trying to create an adaptation of the lib for react-native, and I couldn't understand the purpose while looking at the code.
Sorry for the question unrelated to the development of the library🙏

**Comment by lindell** [comment, marked as answer]
It removes the guard bars: https://github.com/lindell/JsBarcode/wiki/EAN

**Reply by TheBlindHawk** [reply]
Thank you so much for the quick answer!
When I tried it on the codepen there was no difference, could it be a version issue?
(the codepen version seems to be 3.3.20)

Edit: it was a version issue, worked fine once I set the codepen to 3.11 :)
